### PR TITLE
Feature/crear formulario datos usuario #2

### PR DESCRIPTION
**Pull request by jossee0219**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -26,3 +26,86 @@
...
+  background: white;
+  width: 400px;
+  height: 247px;
+  padding-left: 15px;
+  margin-top: 20px;
+  display: inline-block;
+  margin-right: 20px;
+}
+
+.prueba{
```
Comment (serudda) [inline src/App.css]: English please, If you are not using this 'class' please remove it.

File: src/FormUser.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+
+class FormUser extends React.Component{
+    
+    constructor(){
+        super();
+        this.state = { users: [], textn: '', textl: '', texte: '', textp: '', textc: '', texta: ''};
```
Comment (serudda) [inline src/FormUser.js]: Please assign name more descriptive: e.g nameText, lastNameText, etc.  (camel Case format)

File: src/FormUser.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                    value = {this.state.textp}>
+                </input>
+
+                <button className='save' onClick = {this._handleClick}>Save!!</button>
+
+                <ul>
+                    {this.state.users.map(user => (
+                        <div className='container'>
+                            <div key={user.id} className='prueba'>
+                                <table>
```
Comment (serudda) [inline src/FormUser.js]: Table is not recommended for this kind of 'containers', please refactor this using **div**. If it's necessary, add **boostrap 4**.

**Review by serudda — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: src/App.css (hunk 1 above)
Comment (jossee0219) [reply src/App.css]: Done.

File: src/FormUser.js (hunk 2 above)
Comment (jossee0219) [reply src/FormUser.js]: Done.

File: src/FormUser.js (hunk 3 above)
Comment (jossee0219) [reply src/FormUser.js]: Done.

**Review by serudda — APPROVED** [review]
Approved

**Pull request merged**